application/src/main/java/org/togetherjava/tjbot/commands/basic/VcActivityCommand.java

```diff
@@ -223,51 +214,29 @@ private static void handleErrors(@NotNull SlashCommandEvent event,
         logger.warn("Something went wrong in the VcActivityCommand", throwable);
     }
 
-
     /**
-     * This grabs the OptionMapping, after this it <br />
-     * - validates whenever it's within an {@link Integer Integer's} range <br />
-     * - validates whenever it's positive <br />
+     * This grabs the OptionMapping, after this it returns null of the OptionMapping is null, else
+     * it'll return the number option as an Integer
```

Comment:
`if` not `of`. That said, this javadoc rather explains the implementation and not the intention or idea behind the method.

Maybe reword to `Interprets the given option as integer. Throws if the option is not an integer.` And then `@return the option interpreted as `int`, `null` if the option was `null` already.